docs/README.md

```diff
@@ -0,0 +1,52 @@
+# Clay internal documents
+
+It consists of short documents that explain how flows, tools, how processes work on the Clay project.
+
+Documents are not necessarily technical but any document that helps to understand certain motivations or that centralizes information that has a benefit for the entire team.
+
+There are two types of documents, repository infrastructure documents, release cycles or explanations of internal tools, and [RFCs documents](#rfcs) that are used to discuss and centralize "substantial" changes in Clay that reflect on components or documentation that are sourced to users.
+
+## Docs
+
+-   [Developing components](developing_components.md)
+-   [How are clayui.com pages created](creating_pages.md)
+-   [Clay CSS Version History](clay_css_version_history.md)
+
+## RFCs
+
+Clay evolves rapidly and undergoes constant changes as the DXP platform evolves and takes new forms, new components are implemented over time or are evolved to add new behaviors, conversations get lost in various PRs and issues.
+
+However, some of these changes and agreements need to be documented or pin so that we can take them into account before adding new behaviors or helping with decision making, they are easy to find and they are objective and enlightening.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the project.
+
+### When to follow this process
+
+You should consider using this process when you intend to make "substantial" changes to Clay or its documentation. Some examples that have benefited from an RFC are:
+
+-   A new feature that creates a new API surface area that can be common across all components.
+-   The removal of features that already shipped as part of the release channel.
+-   The introduction of a new organization of documents or significant changes.
+
+The RFC process is a great opportunity to draw more attention to your proposal. Even if the proposals appear to be "obvious" they can be improved significantly, since the team can discuss together with other people and they can give their opinion.
+
+The RFC process can also be helpful to encourage discussions about a proposed feature as it is being designed, and incorporate important constraints into the design while it's easier to change, before the design has been fully implemented.
+
+### What is the process
+
+To add a new feature or make a major change to Clay components or change direction, an RFC must first be submitted and that it first merged as a markdown file. Once merged, the RFC is awaiting implementation.
+
+-   Fork this repository
+-   Copy `0000-proposal-template.md` for `proposals/0000-my-feature.md` (`my-feature` is descriptive)
```

Comment:
It might involve less overheard if instead of forking and copying files, they could open an RFC issue. Or maybe "Discussions" would be a worthy feature to try for this?

To enable, go to repo settings
<img width="1018" alt="Screen Shot 2021-06-15 at 4 26 19 PM" src="https://user-images.githubusercontent.com/6843530/122135978-6da34e80-cdf6-11eb-841d-59f0995392ff.png">

Reply:
And an RFC markdown file could be added after the issue is successfully work through.

Reply:
https://docs.github.com/en/discussions/quickstart

Reply:
Hey @bryceosterhaus I think it might make sense and we can try maybe the only thing I think is bad about some of them is just the proposal review flow, for example, in "issues" and "Discussions" I see one more initial step to understand what it is the problem of what we create the solid proposal and start to discuss there. With the flow of PRs is that we can select parts of the text and comment on, how to start a conversation on this subject, if I'm not mistaken I think we can even do with "Discussions" but we would have to copy and start a thread below the comment to keep things organized. I saw an Apple use case with Swift, that before creating an RFC you need to discuss this issue in the forum and the RFC header should link to this discussion, I imagine for example issue #4112, trying to collect more information to get have an overview and then open an RFC to discuss the proposal. Does this make sense to you?

Reply:
> I imagine for example issue #4112, trying to collect more information to get have an overview and then open an RFC to discuss the proposal.

This makes sense, it might be fragmented in the RFC if you are pointing to different issue comments. If the RFC was created through an issue, you could still use #4112 as a starting point and then close the issue and point to the official RFC issue.


> With the flow of PRs is that we can select parts of the text and comment on, how to start a conversation on this subject,

Yeah this is definitely a benefit to a PR. My only thought for not using a PR is that it requires the users to take a few additional steps, _clone_, _fork_, _make changes in editor_, and _send pr_. Whereas if it was all contained in the web via issues, they don't have to take those steps. I do think if we can offer the least path of resistance, we will get more feedback from other teams. I assume other developers are a lot like myself _(lazy)_ and that means the more steps I have to take, I am less likely to do it.

Regardless if it is an issue or PR though, I think the proposal process would be helpful and is a good idea and so go with whatever method you think would be best.

Reply:
Yeah that's true, it might be a bit of friction for some people as it will have to be fork, commit... I think if we want to go with some of them, probably the best scenario would be the "Discussions" for having a better flow to keep conversations organized I think. But anyway I would like to test this with the PR initially and we can add more recommendations to encourage more use.

I imagine who will be using this more would be us than other people are proposing, I would be very happy if someone can contribute to us but I hope it helps us to record the substantial changes and also to think better about them before doing.